### ISSUE TITLE
Allow (driver) services to set paths for specific capabilities

### ISSUE THREAD

**Issue by christian-bromann** [webdriverio/webdriverio#4694]
All browser drivers run on `/` as default path. Having the default path set to `/wd/hub` is a relict from the past and only applicable when using Selenium Standalone. Therefor let's change the default path to `/`.

**Comment by mgrybyk** [comment]
It's not just selenium standalone but everything that is built on top, like appium, selenium grid and some cloud services. Not sure if it makes much sense to do it.

What do you think?

**Comment by christian-bromann** [comment]
> What do you think?

To be honest, I don't have a strong opinion about this at all.

**Comment by mgrybyk** [comment]
The gotcha here is that we can't change path with the service in a launcher part. We can think how to deal with it better in v6

**Comment by christian-bromann** [comment]
Before we can tackle this we need #4721 to be resolved first. Then task of this issue should be to modify our existing services to leverage the result of #4721.

**Comment by christian-bromann** [comment]
Before I can tackle this I would like to get #4926 and #4606 merged otherwise I will run into merge conflicts.

**Comment by erwinheitzman** [comment]
Currently the `6.0.0-alpha.1` throws the following error when using the `selenium-standalone` service.

```
Unable to connect to "undefined:undefined", make sure browser driver is running on that address.
```

**Comment by christian-bromann** [comment]
Fixed in #5032
